cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -557,6 +558,12 @@ func StartControllers(s *options.CMServer, kubeconfig *restclient.Config, rootCl
 		}
 	}
 
+	bootstrap.NewBootstrapSigner(
+		client("bootstrap-signer"),
+		bootstrap.DefaultBootstrapSignerOptions(),
+	).Run()
```

Comment:
Run should be started with the stop chan that is in scope here like other controllers. It should not have an explicit stop method, but should stop when the channel is closed.